resources/views/Checkout/AmazonPayCheckout.twig

```diff
@@ -1,12 +1,12 @@
-{% if services.template.isCurrentTemplate('tpl.checkout') %}
+{% if services.template.isCurrentTemplate('tpl.checkout') or services.template.isCurrentTemplate('tpl.my-account') or services.template.isCurrentTemplate('tpl.confirmation') %}
```

Comment:
Still some code inside which isn't necessary for the confirmation or myaccount page.
I think the best would be to split up this file as well.